canonical-web-and-design/jaas-dashboard
# Responsive cleanup

## Done

Responsive cleanup around the whole app

## QA

- Pull code
- Run `./run clean && ./run serve`
- Open http://0.0.0.0:8036/
- View the app in a variety of screen sizes
- Verify notable improvement

## Details

Fixes #229
